electrumsv/daemon.py

```diff
@@ -312,9 +357,23 @@ def run_cmdline(self, config_options: dict) -> Any:
         result = func(*args, **kwargs)
         return result
 
+    def on_stop(self):
+        if self.rest_server.is_alive:
+            app_state.async_.spawn_and_wait(self.rest_server.stop)
+            self.logger.debug("stopped.")
```

Comment:
This logging needs to be corrected to say what has stopped or shifted to the function top level, because it's not the daemon, it's the daemon if there is a rest server.

Reply:
Ah yes. When I added that if statement there, I didn't keep the indent level the same for the logging at top-level. Next commit will have this fixed.